opentracing-spring-web-handler-interceptor/src/test/java/io/opentracing/contrib/spring/web/servlet/AppConfiguration.java

```diff
@@ -0,0 +1,150 @@
+package io.opentracing.contrib.spring.web.servlet;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.concurrent.Callable;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.context.request.async.DeferredResult;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+import io.opentracing.contrib.web.servlet.filter.TracingFilter;
+import io.opentracing.mock.MockTracer;
+import io.opentracing.mock.TextMapPropagator;
+
+/**
+ * @author Pavol Loffay
+ */
+@Configuration
+@EnableAutoConfiguration
+public class AppConfiguration extends WebMvcConfigurerAdapter {
+
+    public static MockTracer mockTracer = new MockTracer(new TextMapPropagator());
+
+    @Bean
+    public Filter exceptionFilter() {
+        return new ExceptionFilter();
+    }
+
+//    @Bean
+    public FilterRegistrationBean tracingFilterRegistration() {
+        FilterRegistrationBean registrationBean = new FilterRegistrationBean();
+        TracingFilter tracingFilter = new TracingFilter(mockTracer, Arrays.asList(
+                io.opentracing.contrib.web.servlet.filter.SpanDecorator.STANDARD_TAGS),
```

Comment:
This is a left over when I was testing integration with servlet filter. The whole block is removed.